_episodes/03-running-containers.md

```diff
@@ -151,16 +151,17 @@ $ docker run alpine
 ~~~
 {: .language-bash}
 
-Probably nothing! That's because this particular container is designed for you to
+If you never used the *alpine* docker image on your computer, docker told you that it couldn't find the image and had to download it fresh.
+If you did the download before, the command will probably show no output. That's because this particular container is designed for you to
```

Comment:
```suggestion
If you had used the alpine image before, the command will probably show no output. That's because this particular container is designed for you to
```

Reply:
simply `used` instead of `had used` ?

Reply:
Good catch!